toolbox-reference_5b7bdc1d0428631d7a8a2fcc.md

```diff
@@ -289,6 +297,8 @@ which are automatically set in every job environment:
     SSH key that will be used for connecting to the cache server
   (`/home/semaphore/.ssh/semaphore_cache_key`).
 
+Additionally, `cache` uses `tar` to archive a specified directory or file.
```

Comment:
I would rephrase it as (in order to be more generic):

Additionally, `cache` uses `tar` to archive the specified directories and files.

Reply:
Cache store command can be used in the following examples:

- To cache directory we specify directory path. For example, if we want to cache `vendor/bundle` we'd use a command similar to the following:
`cache store gems vendor/bundle`
- On the other hand, we can also cache only a single file:
`cache store gemfile-list Gemfile.lock`

I'm under the impression that stating `files` in this context will create confusion - that one execution of `cache store` can store more files at once.